src/requirements.in

```diff
@@ -42,3 +42,4 @@ pendulum == 2.1.2
 paramiko==2.7.2
 pysftp==0.2.9
 sshtunnel==0.1.4
+flask-openid==1.3.0
```

Comment:
waarom is die ineens hier nodig?

Reply:
See commit 4f213cbd24ca9a43dbf99648c5124986820025b9

I try to always document the why in my commit messages ;-)